task3.py

```diff
@@ -0,0 +1,24 @@
+# Задайте список из вещественных чисел. Напишите программу, которая найдёт разницу между максимальным и минимальным значением дробной части элементов.
+# Пример:
+# - [1.1, 1.2, 3.1, 5, 10.01] => 0.2
+import random
+x = int(input('введите длину списка: '))
+list = []
+
+for i in range(0, x, 1):
```

Comment:
```suggestion
for i in range(x):
```
Старт с 0 и шаг 1 передаются по умолчанию.